packages/components/src/segmented-control/component.tsx

```diff
@@ -51,9 +49,12 @@ function SegmentedControl(
 	const cx = useCx();
 	const containerRef = useRef();
 	const [ resizeListener, sizes ] = useResizeAware();
-
+	const baseId = useInstanceId(
+		SegmentedControl,
+		'segmented-control'
+	).toString();
```

Comment:
is the toString needed?

Reply:
Yes, because `useInstanceId` returns `number|string`, where `useRadioState.baseId` expects a string. By providing the prefix `'segmented-control'` we already have a string but we need to cast it for TS type checking.